python/cuda_cccl/pyproject.toml

```diff
@@ -16,27 +16,49 @@ classifiers = [
   "License :: OSI Approved :: Apache Software License",
 ]
 requires-python = ">=3.9"
+
 dependencies = [
   "numba>=0.60.0",
   "numpy",
-  "cuda-bindings>=12.9.1,<13.0.0",
   "cuda-pathfinder>=1.1.0",
   "cuda-core",
-  "nvidia-cuda-nvrtc-cu12",
-  "nvidia-nvjitlink-cu12",
   "numba-cuda>=0.18.0",
 ]
+
 dynamic = ["version"]
 readme = { file = "README.md", content-type = "text/markdown" }
 
 [project.optional-dependencies]
-test = [
+cu12 = [
+  "cuda-bindings>=12.9.1,<13.0.0",
+  "nvidia-cuda-nvrtc-cu12",
+  "nvidia-nvjitlink-cu12",
```

Comment:
```suggestion
  "cuda-toolkit[nvrtc,nvjitlink]==12.*",
```

Reply:
Can you explain what exactly this does? Does it grab some base CTK + nvrtc and nvjitlink, or _just_ nvrtc and nvjitlink?

Reply:
The latter. This new metapackage supports 11.8, 12.x, and 13.x, allowing us to do a better dependency constraint https://github.com/NVIDIA/cuda-python/issues/853